TOPAS/Plots/2GySOBP/z-plot.py

```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from topas2numpy import BinnedResult
+
+files = []
+files.append(BinnedResult('/home/villads/2022_DCPT_LET/TOPAS/output/protons.592479/Scoring_protonD_ZBox.csv'))
+files.append(BinnedResult('/home/villads/2022_DCPT_LET/TOPAS/output/protons.592479/Scoring_protonLET_ZBox.csv'))
+for data in files:
+    print('{0} [{1}]'.format(data.quantity, data.unit))
+    print('Statistics: {0}'.format(data.statistics))
```

Comment:
```suggestion
    print(f'Statistics: {data.statistics}')
```

better to use f-strings